### PR TITLE
fix(core): add class aliases to prevent name collissions

### PR DESCRIPTION
**Pull request by hypeJunction**
In some early versions of PHP 7 class autoloading is causing conflicts
in names resulting in fatal errors.

### CODE REVIEW

**Comment by jeabakker** [comment]
I think i encountered this ;) PHP < 7.0.13 i believe

File: engine/classes/Elgg/Gatekeeper.php [hunk 1]
```diff
@@ -156,23 +156,11 @@ public function assertAccessibleEntity(ElggEntity $entity, ElggUser $user = null
 			if (!$this->session->getIgnoreAccess() && !$this->access->hasAccessToEntity($entity, $user)) {
 				// user is logged in but still does not have access to it
 				$msg = $this->translator->translate('limited_access');
-				$exception = new EntityNotFoundException($msg);
-				$exception->setParams([
```
Comment (jdalsem) [inline engine/classes/Elgg/Gatekeeper.php]: huh why did you remove this? you added this recently am i right?

Comment (hypeJunction) [reply engine/classes/Elgg/Gatekeeper.php]: Eh, weird. I will have to redo the PR

**Comment by hypeJunction** [comment]
I moved to a different server. There were other issues popping up here and there, so perhaps a more appropriate solution would be to require a higher version of PHP 7.0, e.g. >=7.0.13
